client/asset/btc/btc.go

```diff
@@ -349,7 +349,7 @@ func (d *Driver) Info() *asset.WalletInfo {
 	return WalletInfo
 }
 
-func init() {
+func RegisterDriver() {
 	asset.Register(BipID, &Driver{})
 }
```

Comment:
I'm not sure if because of this change, but the simnet tests stopped working.

Reply:
Likely. What's the error you get? I'll run the simnet tests later today anyways, I expected there might be a need to make some changes to get them working.

Reply:
<details closed>
<summary>error</summary>

```
=== RUN   TestTradeSuccess
2021-10-05 09:18:09.784 [INF] TEST: === SETUP
2021-10-05 09:18:09.784 [INF] CORE:1[DB]: creating new version 5 database
2021-10-05 09:18:09.784 [DBG] CORE:1: Using locale printer for "en-US"
2021-10-05 09:18:09.784 [DBG] CORE:1: new client core created
2021-10-05 09:18:09.784 [INF] TEST: [client 1] Core created
2021-10-05 09:18:09.784 [INF] CORE:1: Starting DEX client core
2021-10-05 09:18:09.784 [INF] CORE:1: Successfully connected to 0 out of 0 DEX servers
2021-10-05 09:18:09.817 [INF] CORE:1: notify: |SUCCESS| (security) Don't forget to back up your application seed - A new application seed has been created. Make a back up now in the settings view.
2021-10-05 09:18:09.817 [INF] TEST: [client 1] Core initialized
    trade_simnet_test.go:200: error starting clients: asset with BIP ID 42 is unknown. Did you _ import your asset packages?
--- FAIL: TestTradeSuccess (0.06s)
```
</details>